04_Chatbot/00_Chatbot_Claude.ipynb

```diff
@@ -156,7 +156,7 @@
     "from langchain.memory import ConversationBufferMemory\n",
     "\n",
     "cl_llm = Bedrock(\n",
-    "    model_id=\"anthropic.claude-v1\",\n",
+    "    model_id=\"anthropic.claude-v2\",\n",
```

Comment:
Do you think we should use the instant model here and a few other places a bit more?

I know in this PR you are just updating v1 to v2, but with the price/performance trade-offs of the instant, I wonder if chat and question answering might be good candidates to move to the instant model.

Let me know what you think.

Reply:
I think it's better for people to start with the bigger/smarter model to see if they can hit their performance bar, as that's often the first blocker. Then they can optimize further from there.